backend/src/test/resources/upload-patients

```diff
@@ -0,0 +1,15 @@
+-----------------------------23343861241310289970349754859
+Content-Disposition: form-data; name="operations"
+
+{"operationName":"UploadPatients","variables":{"patientList":null},"query":"mutation UploadPatients($patientList: Upload!) {\n  uploadPatients(patientList: $patientList)\n}\n"}
+-----------------------------23343861241310289970349754859
+Content-Disposition: form-data; name="map"
+
+{"1":["variables.patientList"]}
+-----------------------------23343861241310289970349754859
+Content-Disposition: form-data; name="1"; filename="test-upload.csv"
+Content-Type: text/csv
+
+LastName,FirstName,MiddleName,Suffix,Race,DOB,biologicalSex,Ethnicity,Street,Street2,City ,County,State,ZipCode,PhoneNumber,employedInHealthcare,residentCongregateSetting,Role,Email,facilityId
+Best,Tim,,,White,5/11/1933,Male,Not_Hispanic,123 Main Street,,Washington,,DC,20008,5656667777,Yes,No,Staff,foo@example.com,
```

Comment:
This doesn't exactly stress-test the upload, but thanks for getting a working example into the source tree!